bindings/nodejs/package.json

```diff
@@ -20,6 +20,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@iota/types": "^1.0.0-beta.15",
+    "@iota/wallet": "^2.0.3-rc.7",
```

Comment:
I'm not a JS expert but it has itself in dependencies ?

Reply:
Damnit how did that happen 🤣